karnbot/bot.py

```diff
@@ -20,7 +23,13 @@ async def on_ready():
     print(f"Bot connected as {bot.user}")
     # Setup test channel
     global test_channel
-    test_channel = bot.get_channel(780611534833188905)
+
+    # Setup Config to get test channel details
+    config = configparser.ConfigParser()
+    config.read("config.ini")
+    channel_id = config["Discord"]["TestChannel"]
```

Comment:
Why are we using an entire new config file instead of reusing `dotenv` above, which also works via Environment variables. Its a better solution.